python/mister_webhooks/types.py

```diff
@@ -55,7 +55,7 @@ class Encoding(Enum):
 class KafkaMessageEnvelopeV1(TypedDict):
     encoding: Literal["JSON", "CBOR"]
     method: Literal["GET", "HEAD", "POST", "PUT", "DELETE", "PATCH"]
-    headers: List[Tuple[str, List[bytes]]]
+    headers: Dict[str, List[bytes]]
```

Comment:
Do you have any tests that would've caught the original problem? This is one of the few times I think OpenAutocomplete would shine.